Distraction Shield/background.js

```diff
@@ -79,22 +79,13 @@ removeWebRequestListener = function() {
 intercept = function(details) {
     interception.incrementInterceptionCounter(details.url);
     interception.addToInterceptDateList();
-    var redirectLink;
-    var params;
-    if (!auth.sessionAuthentic) {
-        redirectLink = chrome.extension.getURL('loginPage/login.html');
-        params = "?forceLogin=" + zeeguuExLink;
-    } else {
-        redirectLink = zeeguuExLink;
-        params = "?sessionID=" + localSettings.getSessionID();
-    }
-    params = params+"&redirect="+details.url;
-
+    var redirectLink = zeeguuExLink;
+    var params = "?redirect="+details.url;
     return {redirectUrl: redirectLink + params};
 };
 
 handleInterception = function(details) {
-    if (localSettings.getState() == "On") {
+    if (localSettings.getState() == "On" && details.url.indexOf("/oauth") < 0) {
```

Comment:
That's a confusing new condition. I get that it's authentication related, but still..